### PR TITLE
Use default destination

### PR DESCRIPTION
**Pull request by syucream**
We don't have `/index.html` and https://github.com/dmm-com/airone/deployments/github-pages navigates https://dmm-com.github.io/airone/. But currently we have pages under `docs` so accessible page URL is https://dmm-com.github.io/airone/docs/
The `docs/` directory is necessary? If not so, it looks better to remove the level. How about this?

### CODE REVIEW

**Comment by syucream** [comment]
The error on staticchecks should be fixed by https://github.com/dmm-com/airone/pull/920

**Comment by hinashi** [comment]
helm is published in index.yaml.
There is a problem that deploying hugo deletes the index.yaml if `/airone/` is the destination.
It may be possible to avoid it with options. exclude_assets? keep_files?
https://github.com/peaceiris/actions-gh-pages

Also, the following will need to be changed.
https://github.com/dmm-com/airone/blob/843e4f0c2f15d254b15ade3e4b051d2aa22a39e6/docs/config.yaml#L2

**Comment by syucream** [comment]
fixed!

**Review by hinashi — APPROVED** [review]
LGTM!

File: .github/workflows/release-hugo.yml [hunk 1]
```diff
@@ -24,4 +24,4 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./docs/public
-          destination_dir: docs
+          keep_files: true
```
Comment (userlocalhost) [inline .github/workflows/release-hugo.yml:27]: Thank you for solving following requirement with elegant solution 👍 
https://github.com/dmm-com/airone/pull/921#issuecomment-1687599150

**Review by userlocalhost — APPROVED** [review]
Thank you very much for its great work!!

**Pull request merged**